lesson03.find-a-pair/scripts.js

```diff
@@ -0,0 +1,248 @@
+class Cards {
+    constructor(name, color, selected) {
+        this.name = name;
+        this.color = color;
+        this.selected = false;
+    }
+}
+class GroupOne extends Cards {
+}
+class GroupTwo extends Cards {
+}
+const card_1 = new GroupOne('1', 'orange', 'false');
+const card_2 = new GroupOne('2', '#f25', 'false');
+const card_3 = new GroupOne('3', 'green', 'false');
+const card_4 = new GroupOne('4', 'blue', 'false');
+const card_5 = new GroupOne('5', 'yellow', 'false');
+const card_6 = new GroupOne('6', 'purple', 'false');
+const card_7 = new GroupOne('7', 'brown', 'false');
+const card_8 = new GroupOne('8', 'red', 'false');
+const card_9 = new GroupTwo('9', 'purple', 'false');
+const card_10 = new GroupTwo('10', 'brown', 'false');
+const card_11 = new GroupTwo('11', 'red', 'false');
+const card_12 = new GroupTwo('12', 'yellow', 'false');
+const card_13 = new GroupTwo('13', 'green', 'false');
+const card_14 = new GroupTwo('14', 'orange', 'false');
+const card_15 = new GroupTwo('15', '#f25', 'false');
+const card_16 = new GroupTwo('16', 'blue', 'false');
+"use strict";
+let allSections = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16'];
+let numOfClick = 0;
+
+let card1 = `<section class="cards" onclick="openCardFun('1')"  id="1"></section>`;
+let card2 = `<section class="cards" onclick="openCardFun('2')"  id="2"></section>`;
+let card3 = `<section class="cards" onclick="openCardFun('3')"  id="3"></section>`;
+let card4 = `<section class="cards" onclick="openCardFun('4')" id="4"></section>`;
+let card5 = `<section class="cards" onclick="openCardFun('5')" id="5"></section>`;
+let card6 = `<section class="cards" onclick="openCardFun('6')" id="6"></section>`;
+let card7 = `<section class="cards" onclick="openCardFun('7')" id="7"></section>`;
+let card8 = `<section class="cards" onclick="openCardFun('8')" id="8"></section>`;
+let card9 = `<section class="cards" onclick="openCardFun('9')" id="9"></section>`;
+let card10 = `<section class="cards" onclick="openCardFun('10')" id="10"></section>`;
+let card11 = `<section class="cards" onclick="openCardFun('11')" id="11"></section>`;
+let card12 = `<section class="cards" onclick="openCardFun('12')" id="12"></section>`;
+let card13 = `<section class="cards" onclick="openCardFun('13')" id="13"></section>`;
+let card14 = `<section class="cards" onclick="openCardFun('14')" id="14"></section>`;
+let card15 = `<section class="cards" onclick="openCardFun('15')" id="15"></section>`;
+let card16 = `<section class="cards" onclick="openCardFun('16')" id="16"></section>`;
```

Comment:
Assign event in a proper matter (like `AddSeventListener`) and use value / `data-attribute`